### DISCUSSION TITLE
how to reset on current elapsed goal time?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [poetaman/arttime#62] [Q&A]
I am using `arttime -k timer.pomodoro4etc`, but sometime i miss the current goal because i took too much time comming back. How to only reset current goal elapsed time to zero?

**Comment by poetaman** [comment]
@KAGEYAM4 Thanks for the question. Can you please explain what you mean by:

> but sometime i miss the current goal because i took too much time comming back

arttime is designed primarily for those who spend their day looking at their computer screens. If you step away from your computer while a pomodoro program is active, then you can pause the entire pomodoro program (current goal gets paused) before leaving computer. To do that press `p` and answer `y`. When you come back to the computer, press `p` again followed by `y` to restart the pomodoro program where you left. If you worked for 10 minutes in a pomodoro work sprint before leaving, you will pick the current goal at 10 minutes again.

At the moment there isn't a way to reset current goal elapsed time to zero. Let me know whether the pausing solves your case.

**Comment by cameronj86** [comment]
Low priority, but also something I'm interested in for when I get heavily distracted mid-pomodoro and want to start w/ a fresh 25m, but have my `keypoems` already set consecutively.

**Reply by poetaman** [reply]
Ok thanks for your feedback, looks like this should be in my todo list! Though yeah, it's low priority as I am busy with other things atm.
